ember-core/shared/src/main/scala/org/http4s/ember/core/h2/H2Frame.scala

```diff
@@ -0,0 +1,885 @@
+/*
+ * Copyright 2019 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s.ember.core.h2
+
+import cats.syntax.all._
+import scodec.bits._
+
+private[ember] sealed trait H2Frame
```

Comment:
Yeah, my eyes glazed over on this file.  I'm trusting there are tests to give confidence there aren't errors in all these constants.